src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets

```diff
@@ -76,8 +88,11 @@
         <Name>$(GitHubRepositoryName)</Name>
         <Version>1.0.0</Version>
         <ExactVersion>1.0.0</ExactVersion>
-        <Sha>@(RootRepoCommitSha)</Sha>
+        <!-- Pin source-build to a specific commit to provide stabilization -->
+        <!-- <Sha>@(RootRepoCommitSha)</Sha> -->
```

Comment:
Wouldn't the commit be a moving target for main ?

Reply:
I'm not sure, @MichaelSimons ?

Reply:
Yes, source-build will utilize a LKG until 7.0 gets to a stable point.